### ISSUE TITLE
Missing protocols 79998 and 80188

### ISSUE THREAD

**Issue by amartin916** [icza/s2prot#25]
They are both the same as 78285 (as far as I can tell)

**Comment by icza** [comment]
Added. Thanks for reporting.

**Comment by oze4** [comment]
I am still getting this error - should this be resolved?

I am using [example code](https://github.com/icza/s2prot/blob/master/_example/s2prot.go) from this repo, with one of my replays.

```
Version: 4.12.0.80188
Base build: 80188
panic: Unknown base build!
```

**Comment by icza** [comment]
Yes, this is resolved. Please update your `s2prot` module to the master branch like this:

    go get -u github.com/icza/s2prot@master

(Assuming you're using go modules.)

**Comment by oze4** [comment]
It's odd because I created a new pkg today and installed s2prot today (I assumed it would use the newest version?)

At any rate, thank you for your response! @icza

**Comment by icza** [comment]
It won't use the latest because there are version released pass beyond `v1.0.0`, and the latest commits on `master` are not yet included in the releases. So go modules will use the latest stable version `v1.x` version by default.
